docs/README.md

```diff
@@ -3,83 +3,101 @@
 
-| Concept / Feature | Description                           |
-|:-------------------|:---------------------------------------|
-| Client(s)| Browser, App or API Client that sends requests to Couper.|
-| Web Serving|Couper supports file serving and Web serving for SPA assets.|
-|API| Configuration block that bundles endpoints under a certain base path.|
-|Access Control| Couper handles access control for incoming client requests.|
-|Endpoint| Configuration block that specifies how (and if) requests are sent to backend service(s) after they reach Couper.|
-|Backend|Configuration block that specifies the connection to a local/remote backend service.|
-|Logging|Couper provides standard logs for analysis and monitoring.|
-|Backend Service(s)|External API or micro services where Couper fetches data from.|
+## Core concepts
 
-## Configuration file <a name="conf_file"></a>
+![overview](./overview.png)
 
-### Syntax <a name="syntax"></a>
+| Concept / Feature  | Description |
+|:-------------------|:------------|
+| Client(s)          | Browser, App or API Client that sends requests to Couper. |
+| Web Serving        | Couper supports file serving and web serving for SPA assets. |
+| API                | Configuration block that bundles endpoints under a certain base path. |
+| Access Control     | Couper handles access control for incoming client requests. |
+| Endpoint           | Configuration block that specifies how (and if) requests are sent to backend service(s) after they reach Couper. |
+| Backend            | Configuration block that specifies the connection to a local/remote backend service. |
+| Logging            | Couper provides standard logs for analysis and monitoring. |
+| Backend Service(s) | External API or micro services where Couper fetches data from. |
 
-The syntax for Couper's configuration file is [HCL 2.0](https://github.com/hashicorp/hcl/tree/hcl2#information-model-and-syntax), a configuration language by HashiCorp.
+## Configuration file
 
-### File name <a name="file_name"></a>
+### Syntax
 
-The file-ending of your configuration file should be `.hcl` to have syntax highlighting within your IDE.
+The syntax for Couper's configuration file is
+[HCL 2.0](https://github.com/hashicorp/hcl/tree/hcl2#information-model-and-syntax),
+a configuration language by HashiCorp.
 
-The `filename` defaults to `couper.hcl` in your working directory. This can be changed with the `-f` command-line flag.
-With `-f /opt/couper/my_conf.hcl` couper changes the working directory to `/opt/couper` and loads `my_conf.hcl`.
- 
+### File name
 
-### Basic file structure <a name="basic_conf"></a>
-Couper's configuration file consists of nested configuration blocks that configure web serving and routing of the gateway. Access control is controlled by an `access_control` attribute that can be set for blocks. 
+The file name of your configuration file should be `couper.hcl` to have syntax
```

Comment:
This is a textual change which should be reverted, the IDE applies the highlighting based on the file-ending, not the filename. This applies to all IDEs which supports `HCL`.